specification/cosmos-db/resource-manager/Microsoft.DocumentDB/2015-04-08/cosmos-db.json

```diff
@@ -921,6 +1327,277 @@
           "description": "URL to get the next set of operation list results if there are any."
         }
       }
+    },
+    "UsagesResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Usage"
+          },
+          "description": "The list of usages for the database. A usage is a point in time metric"
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "description": "The response to a list usage request."
+    },
+    "Usage": {
+      "properties": {
```

Comment:
Can the properties in here be marked as `readOnly`? So far, they're only used in responses it seems. 🙂

Reply:
fixed